### ISSUE TITLE
Using Ping Keybinds In MineMenu Breaks Them

### ISSUE THREAD

**Issue by Brittank88** [GirafiStudios/MineMenu#95]
`Minecraft 1.16.4`
`Forge 35.1.37`
`MineMenu 1.8.7`
`Ping 1.6.9`
My [latest.log](https://hatebin.com/zrmojfzofw) (there is no crash or error I can see...)

Have just reinstalled Ping and MineMenu. The Ping individual keybinds work totally fine, up until I use any single one of them inside of MineMenu. Once I do this, they ALL stop working whatsoever, both inside and outside of MineMenu.

Following this, MineMenu then also refuses to reload and accept (correct!) manual changes to `menu.json`.

From previous experience, Ping refuses to work even if you disable MineMenu and relaunch - you have to literally reinstall Ping before it starts working again.

TL;DR trying to use Ping in MineMenu breaks both mods, badly.

**Comment by GirafiStudios** [comment]
Pretty sure I fixed this a while back, but just forgot to close it :) 
If this still occurs in 1.18, please report it again and refer to this issue.
